data-otservbr-global/scripts/raids/thais/rats.lua

```diff
@@ -10,7 +10,7 @@ local raid = Raid("thais.rats", {
 	minGapBetween = "36h",
 })
 
-raid:addBroadcast("Rat Plague in Thais!"):autoAdvance("5s")
+raid:addBroadcast("Rat Plague in Thais!", WEBHOOK_COLOR_RAID, "Incoming Raid!"):autoAdvance("5s")
```

Comment:
```suggestion
raid:addBroadcast("Rat Plague in Thais!", { type = WEBHOOK_COLOR_RAID, title = "Incoming Raid!" }):autoAdvance("5s")
```

Reply:
Can you really leave out 'type' as input when calling addBroadcast?

Reply:
Oh, I guess not if you're pssing in webhook stuff

Reply:
I both like and dislike the solution since I have to pass type twice in this case.

Reply:
I'd add those webhook defaults to the Raid class, so you won't have to do anything here in the raid itself

Reply:
I could do it as well. If I understand you correct we could just override the inherited function from the encounter class and extend it with the extra functionality.

Reply:
Yes, that

Reply:
Going to bed now but will fix tomorrow morning be4 work :-)